src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(String fruit, int quantity) {
+        Storage.storage.put(fruit, quantity);
```

Comment:
Directly manipulating the storage map can lead to data inconsistency. Consider encapsulating the logic for updating storage, such as checking if the fruit already exists and then updating or putting the new quantity.